ballerina/tests/parse_bytes_with_projection_options.bal

```diff
@@ -23,23 +23,23 @@ final Options & readonly options1 = {
     allowDataProjection: {
         nilAsOptionalField: true,
         absentAsNilableType: false,
-        strictTupleOrder: false
+        enableYamlStreamReorder: false
```

Comment:
According to the name this should be set to `true`. Right? correct me if I'm wrong